File: src/ios/Sources/AnsightCore/PairingProtocolDefaults.swift
import CryptoKit
import Foundation

public enum PairingProtocolDefaults {
    public static let discoveryPort = 45123
    public static let developerDiscoveryPort = 46123
    public static let webSocketPort = 45124
    public static let webSocketPath = "/ws"

    static let localDiscoveryPorts = [
        discoveryPort,
        developerDiscoveryPort,
    ]
}

File: src/dotnet/Ansight.Core/Pairing/PairingProtocolDefaults.cs
namespace Ansight.Pairing;

/// <summary>
/// Default transport values used by the Ansight pairing protocol.
/// </summary>
public static class PairingProtocolDefaults
{
    /// <summary>
    /// Default UDP discovery port exposed by the host.
    /// </summary>
    public const int DiscoveryPort = 45123;

    /// <summary>
    /// Alternate UDP discovery port exposed by a source-built developer Studio.
    /// </summary>
    public const int DeveloperDiscoveryPort = 46123;

    /// <summary>
    /// Default WebSocket port used for live pairing sessions.
    /// </summary>
    public const int WebSocketPort = 45124;

    /// <summary>
    /// Default WebSocket path used for live pairing sessions.
    /// </summary>
    public const string WebSocketPath = "/ws";
}
